File: Server/Poker-Library/Models/WorkItem.cs
using System;
using Newtonsoft.Json;

namespace Poker.Library.Models
{
    public class WorkItem
    {
        [JsonProperty("key")]
        public Guid Key { get; set; } = Guid.NewGuid();

        [JsonProperty("title")]
        public string Title { get; set; } = "";

        [JsonProperty("body")]
        public string Body { get; set; } = "";

        [JsonProperty("active")]
        public bool Active { get; set; } = false;
    }
}

File: Client/src/models/WorkItem.ts
import { isNothing } from "../utilitites/isNothing";


export interface IWorkItem {
    key: string;
    title: string;
    body: string;
    active: boolean;
}

export default class WorkItem implements IWorkItem {
    public key: string;
    public title: string;
    public body: string;
    public active: boolean;

    constructor(json: IWorkItem = null) {
        if (isNothing(json) === true) return;
        
        this.key = json.key;
        this.title = json.title;
        this.body = json.body;
        this.active = json.active ?? false;
    }
}